protocol/bc/txtransaction.go

```diff
@@ -26,8 +49,8 @@ func ComputeOutputID(sc *SpendCommitment) (h Hash, err error) {
 	return h, nil
 }
 
-// ComputeTxHashes returns all hashes needed for validation and state updates.
-func ComputeTxHashes(oldTx *TxData) (hashes *TxHashes, err error) {
+// TxHashes returns all hashes needed for validation and state updates.
+func ComputeTxEntries(oldTx *TxData) (txEntries *TxEntries, err error) {
```

Comment:
Should this be named something like `Convert` or `Map` or `TxEntriesFromTxData`? Am I right to expect eventually this function will disappear?